include/api.php

```diff
@@ -1485,6 +1485,71 @@ function api_users_search($type)
 /// @TODO move to top of file or somewhere better
 api_register_func('api/users/search', 'api_users_search');
 
+/**
+ * Returns statuses that match a specified query.
+ *
+ * @see https://developer.twitter.com/en/docs/tweets/search/api-reference/get-search-tweets
+ *
+ * @param string $type Return format: json, xml, atom, rss
+ *
+ * @return array|string
+ * @throws UnauthorizedException
+ * @throws BadRequestException
+ */
+function api_search($type)
+{
+	$data = array();
+
+	if (x($_REQUEST, 'q')) {
```

Comment:
You could throw the exeception if "q" was not set at the beginning. So you don't need to indent the rest.